src/helpers/format-duration.test.js

```diff
@@ -0,0 +1,14 @@
+import { formatDuration } from './format-duration';
+
+test.each`
```

Comment:
this use of template literal is amusing! :) you have to deal with string formatting (e.g. positioning pipes in the middle) but it's nice to get column headings in test name, thanks for showing me this!

two issues:
1) tests typically include unexpected inputs as well, here at least `undefined`/`null`
2) 3600000 -> '0:00' why? I would expect '60:00'

congratulations on writing first test in this project! :fireworks:

Reply:
> this use of template literal is amusing! :) 

it is :)

> 1. tests typically include unexpected inputs as well, here at least `undefined`/`null`

right, forgot about it

> 2. 3600000 -> '0:00' why? I would expect '60:00'

yeah, I had some doubts here:
1. do we have to care about such big numbers if we don't plan them (and they depend strictly on our decision)
2. if we did decide on such long duration I would rather go for 1:00:00 than 60:00 (and this is what the function actually does at the moment with minutes, it just does not care about hours)

Reply:
so I suggest removing this test case
we don't handle "big values" and we don't have to
test cases serve also as some kind of documentation
for me "3600000 -> '0:00'" is a bug (but it's irrelevant to us) so it would be weird to include it into tests (including in tests = saying "it should work this way")

Reply:
after some considerations I chose to modify the function so that it returns '60:00' and so on. I think it's more intuitive in this case.

I added throwing error (+ additional tests) but I don't know if it ok 🤔